be/src/storage/memtable.h

```diff
@@ -18,6 +18,8 @@ class TabletSchema;
 
 namespace vectorized {
 
+enum class FlushChunkState { original, upserts, deletes, mixed };
```

Comment:
What's the original is meaning?

Reply:
unknown is a better name

Reply:
it mean that there is no flush before, it is first flush.

Reply:
Original seems to have no correspondence to no flush.

Reply:
good idea, changed